includes/_footer.htm

```diff
@@ -329,7 +329,13 @@
 				var docsHandle = document.querySelector('.toc-handle');
 
 				function toggleDocsWidth() {
-					docsContainer.classList.toggle('expanded');
+					let isExpanded = docsContainer.classList.toggle('expanded');
+					// change the arrow direction based on the view state is expanded or not
+					if (isExpanded) {
```

Comment:
This is technically only working because JavaScript is lax on types and the `DOMTokenList` returned by  `classList.toggle` is empty resulting in a falsy value. We should probably do `isExapnded.contains('expanded')` as the condition to be more explicit.